tests/CocktailParty.tex

```diff
@@ -9,46 +9,45 @@ \subsection{Focus}
 \subsection{Setup}
 \begin{itemize}
 	\item \textbf{Party room}: any (large) room inside the apartment when normally a party would be held.
-	\item \textbf{Guests:} At least five people are distributed in a predefined \quotes{party room} either sitting or standing, some of them forming groups of 2 or 3 people, from which at least one is sitting. Three of the guests have drink orders assigned by the referees. The sitting person always has an assigned drink.
-	\item \textbf{Bar:} The bar is any flat surface where objects can be placed, in a room other than the \quotes{party room}. All available beverages are on top of the bar for the robot can see them.
-	\item \textbf{Barman:} The Barman bay be standing either behind the bar or next to it, depending on the arena setup.
+	\item \textbf{Guests:} At least five people are distributed in a predefined \quotes{party room} either sitting or standing.
```

Comment:
This can be shortened.

```\item \textbf{Guests:} There are at least five people in the \textit{party room}, either sitting or standing. Three of the guests have assigned orders.```

Reply:
shortened from  17 words 123 characters  to  23 words 151 characters?